packages/react-native-select-pro/src/state/reducer.ts

```diff
@@ -78,6 +78,11 @@ export const reducer = <T>(state: State<T>, action: ActionType<T>): State<T> =>
                 ...state,
                 openedPosition: { ...state.openedPosition, ...action.payload },
             };
+        case 'updateOptions':
```

Comment:
can you set the same name for it as you have said in docs? `reinitializeOptions`